torch/onnx/utils.py

```diff
@@ -471,6 +471,21 @@ def _get_example_outputs(model, args):
     return example_outputs
 
 
+_qtype_vtype_map = {torch.quint8: torch.uint8, torch.qint8: torch.int8, torch.qint32: torch.int32, torch.quint4x2: torch.int8}
+
+
+def unpack_quantized_tensor(value):
+    if getattr(value, "dtype", None) in _qtype_vtype_map:
```

Comment:
I think this would be a bit clearer as:
`if isinstance(value, torch.Tensor) and value.dtype in _qtype_vtype_map:`